sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinBridgeEventHandlerImpl.java

```diff
@@ -160,6 +161,13 @@ public void onAudioTrackSelected(ReadableMap parameters) {
     _player.setUserDefinedAudioTrack(parameters.getString("audioTrack"));
   }
 
+  @Override
+  public void onPlaybackSpeedRateSelected(ReadableMap parameters) {
+    Dynamic dynamic = parameters.getDynamic(PLAYBACK_SPEED_RATE);
+    String speed = dynamic.asString();
+    _player.setSelectedPlaybackSpeed(Float.parseFloat(speed));
```

Comment:
use `OoyalaPlayer.setPlaybackSpeed(float)` instead now and we can remove `setSelectedPlaybackSpeed`